pkgs/tools/misc/rmlint/default.nix

```diff
@@ -59,7 +61,9 @@ stdenv.mkDerivation rec {
 
   # this doesn't seem to support configureFlags, and appends $out afterwards,
   # so add the --without-gui in front of it
-  prefixKey = lib.optionalString (!withGui) " --without-gui " + "--prefix=";
+  prefixKey = lib.optionalString (!withGui) " --without-gui "
+    + lib.optionalString (stdenv.isDarwin) " --without-gettext "
```

Comment:
```suggestion
  prefixKey = lib.optionalString !withGui " --without-gui "
    + lib.optionalString stdenv.isDarwin " --without-gettext "
```

Reply:
Note: the parentheses around `!withGui` are required due to the `!`